src/integration/preparePlaceholders.js

```diff
@@ -83,43 +83,11 @@ function moveContentAsyncFile(docuLocation, fileLocation, phLocation, topicDetai
 }
 
 //replace during move of placeholders
-function replaceAndMove(src, dest, topic, next) {
+function replaceAndMove(src, dest, next) {
 
   if(!src || !dest) return next(`Unable to perform operation on topic ${topic.name} because of wrong src: ${src} or dest: ${dest} value`);
 
-  const internal = (dest.indexOf('internal') !== -1),
-    collectionName = internal ? topic.collectionNameInternal : topic.collectionName,
-    releaseNotesPH = internal ? 'internalReleaseNotes' : 'releaseNotes',
-    collectionRN = internal ? 'internalPosts' : 'posts',
-    releaseNotesLayoutPH = internal ? 'internal_release_notes' : 'release_notes',
-    name = topic.name,
-    shortName = topic.shortName,
-    version = topic.version,
-    isLatest = topic.latestVersion ? 'latest' : topic.version,
-    baseUri = topic.baseUri,
-    isInternalUrlOtherPH = internal ? 'internal/' : '',
-    isInternalUrlPH = internal ? '/internal' : '',
-    isInternalPH = internal ? 'Internal' : '',
-    collectionAPIConsoles = internal ? 'internalApiConsoles' : 'apiconsoles',
-    lengthPH = internal ? '4' : '3',
-    internalPH = internal ? 'internal_' : '';
-
   gulp.src(src)
-    .pipe(replace('collectionNamePH', collectionName))
-    .pipe(replace('releaseNotesPH', releaseNotesPH))
-    .pipe(replace('collectionRN', collectionRN))
-    .pipe(replace('releaseNotesLayoutPH', releaseNotesLayoutPH))
-    .pipe(replace('serviceNamePH', name))
-    .pipe(replace('serviceShortNamePH', shortName))
-    .pipe(replace('serviceVersionRAML', version))
-    .pipe(replace('serviceVersionPH', isLatest))
-    .pipe(replace('serviceUri', baseUri))
-    .pipe(replace('isInternalUrlOtherPH', isInternalUrlOtherPH))
-    .pipe(replace('isInternalUrlPH', isInternalUrlPH))
-    .pipe(replace('isInternalPH', isInternalPH))
-    .pipe(replace('collectionAPIConsolesPH', collectionAPIConsoles))
-    .pipe(replace('lengthPH', lengthPH))
-    .pipe(replace('internalPH', internalPH))
     .pipe(gulp.dest(dest))
```

Comment:
Maybe it would be good to use copyFileAsync? It is not necessary, but we usually use it when files are not modified?